material-overrides/assets/stylesheets/kluster.css

```diff
@@ -1,91 +1,103 @@
 :root {
   /* General styles */
-  --border-radius: 0.61rem;
-
-  --md-border-radius: 1.3em;
-  --md-border-width: 1px;
+  --border-radius: 12px;
+  --border-width: 1px;
+  --border-color: var(--primary-green-500);
 
   /* Buttons styles */
-  --feat-button-radius: 0.63em;
-  --feat-button-height: 45px;
-  --feat-button-padding: 0.4rem 1.2rem;
-  --feat-button-width: 1px;
+  --button-radius: 8px;
+  --button-height: 45px;
+  --button-padding: 0.4rem 1.2rem;
+  --button-border: var(--border-width) solid var(--light);
+
+  /* Cards styles general*/
+  --card-border: var(--border-width) solid var(--primary-green-300);
```

Comment:
These colors might change when we go to light mode so they don't really belong in the `:root {`, it makes more sense to put them in `:root,
[data-md-color-scheme='default'] {` 

The things that I typically leave in `:root` are things that are applicable regardless of the theme being used, so for example, the border radius and width and all of the colors. So that just needs to be cleaned up and this will make it easier when you go to add light mode